VAMobile/e2e/tests/VaccineRecords.e2e.ts

```diff
@@ -0,0 +1,53 @@
+import { expect, by, element } from 'detox'
+import { loginToDemoMode, checkImages, openHealth, openVaccineRecords } from './utils'
+
+export const VaccinesE2eIdConstants = {
+  VACCINE_1_ID: 'COVID-19 vaccine January 14, 2021',
+  VACCINE_3_ID: 'FLU vaccine May 10, 2018',
+  VACCINE_5_ID: 'PneumoPPV vaccine April 28, 2016',
+  VACCINE_6_ID: 'FLU vaccine April 28, 2016',
+}
+
+beforeAll(async () => {
+  await loginToDemoMode()
+  await openHealth()
+  await openVaccineRecords()
+})
+
+describe('Vaccine Records Screen', () => { 
+  it('should show vaccine records list content', async () => {
+    await expect(element(by.text('VA vaccines'))).toExist()
+    await expect(element(by.id(VaccinesE2eIdConstants.VACCINE_1_ID))).toExist()
+    var defaultVaccineTemplate = await element(by.id(VaccinesE2eIdConstants.VACCINE_1_ID)).takeScreenshot('defaultVaccineTemplate')
+    checkImages(defaultVaccineTemplate)
```

Comment:
Cool, so we can check a screenshot against the stored screenshot? It's just the entire screen that doesn't work since the clock changes?